coeus-python/src/parse.rs

```diff
@@ -85,6 +107,32 @@ impl AnalyzeObject {
             .collect()
     }
 
+    pub fn get_file(&self, name: &str) -> PyObject {
+        let gil = Python::acquire_gil();
+        let py = gil.python();
```

Comment:
Add `py: Python` as the second argument like in `VmResult::get_value` or `Instruction::execute`